spec/checks/checks_shared_examples.rb

```diff
@@ -0,0 +1,63 @@
+require_relative '../spec_helper'
```

Comment:
I've sliced the original commit into two, so now 6766bdd will contain the refactor of the common tests to a shared file 👍